content/graph/DFSMatching.h

```diff
@@ -5,37 +5,35 @@
  * Source:
  * Description: This is a simple matching algorithm but should
  * be just fine in most cases. Graph $g$ should be a list of
- * neighbours of the left partition. $n$ is the size of the left
- * partition and $m$ is the size of the right partition.
- * If you want to get the matched pairs, $match[i]$ contains
+ * neighbours of the left partition, and $btoa$ should be
+ * a vector with size equal to the size of the right partition.
+ * If you want to get the matched pairs, $btoa[i]$ contains
```

Comment:
This wording is fluffy, let's rewrite it as "Returns the size of the matching. $btoa[i]$ will contain the match ..."